### ISSUE TITLE
Entware Flags

### ISSUE THREAD

**Issue by AntteC** [Waujito/youtubeUnblock#102]
Кинетик 1811 (OS 4.1.7), Entware во внутренней памяти.
Прошу подсказать как менять флаги для запуска на Entware (насколько я понял в README описано для OpenWRT) - на провайдере МФ в дефолтном варианте ютуб периодически фризит (при использовании GoodbuyDPI(для Win)/zapret(для Linux) такого нет при дефолтных флагах.

Так же есть проблема аналогичная https://github.com/Waujito/youtubeUnblock/issues/97 
которая решается загрузкой модулей xt_NFQUEUE.ko и xt_connbytes.ko и перезагрузкой S91youtubeUnblock

**Comment by Waujito** [comment]
Чтобы узнать конкретную причину, вам необходимо будет добавить правила iptables и запустить анблок вручную /opt/bin/youtubeUnblock, посмотреть идут ли логи. Возможно, проблема в kyber, его надо будет отключить в chrome://flags

**Comment by snarknn** [comment]
Вопрос похожий поэтому допишу сюда
Есть youtubeUnblock свежий, установленный на entware на keenetic giga
ютуб разблокировался, все прекрасно но хочется большего

в связи с этим: 
1) как посмотреть какие настройки включены, какие домены в списке разблокируемых?
2) что имеенно надо дописать в ARGS чтобы разблочить, например фейсбук и звонки в тг/воццап?
попробовал написать
`ARGS="--fbegin --sni-domains=facebook.com,fb.com,fbcdn.net,instagram.com,cdninstagram.com"`
но фейсбук не разблочился

ЧЯДНТ?

PS: После обновления ARGS достаточно сделать `/opt/etc/init.d/S91youtubeUnblock restart` или надо что-то еще?

**Comment by dmikhalsky** [comment]
аналогичная проблема. Хочу расширить конфиг. В ARGS пытался прописать файл со списком доменов, но они не подцепляются. 
Ютуб разблокируется, с ним проблем нет.
При запуске пишет загадочный флаг --sni-domains=<trie of 135 vertexes>. Что это?

**Comment by sg7on** [comment]
> аналогичная проблема. Хочу расширить конфиг. В ARGS пытался прописать файл со списком доменов, но они не подцепляются. Ютуб разблокируется, с ним проблем нет. При запуске пишет загадочный флаг --sni-domains=<trie of 135 vertexes>. Что это?

аналогично
добавил адрес, перезагрузил, домен не разблокировался

**Comment by dmikhalsky** [comment]
> > аналогичная проблема. Хочу расширить конфиг. В ARGS пытался прописать файл со списком доменов, но они не подцепляются. Ютуб разблокируется, с ним проблем нет. При запуске пишет загадочный флаг --sni-domains=<trie of 135 vertexes>. Что это?
> 
> аналогично добавил адрес, перезагрузил, домен не разблокировался

в другом issue прочитал, что первый флаг должен быть --fbegin и потом --sni-domains. Домены все равно не разблокируются, но хотя бы при запуске появляется второй набор флагов

**Comment by sg7on** [comment]
> > аналогично добавил адрес, перезагрузил, домен не разблокировался
> 
> в другом issue прочитал, что первый флаг должен быть --fbegin и потом --sni-domains. Домены все равно не разблокируются, но хотя бы при запуске появляется второй набор флагов

установил строку
ARGS="--sni-domains=all --exclude-domains=.ru --quic-drop --fbegin --fend --fbegin --udp-dport-filter=50000-50099 --tls=disabled"
из параллельного топика по разблокировке discord

при старте отчёт по флагам выглядит очень странно
жду ответа там

**Comment by dmikhalsky** [comment]
> > > аналогично добавил адрес, перезагрузил, домен не разблокировался
> > 
> > 
> > в другом issue прочитал, что первый флаг должен быть --fbegin и потом --sni-domains. Домены все равно не разблокируются, но хотя бы при запуске появляется второй набор флагов
> 
> установил строку ARGS="--sni-domains=all --exclude-domains=.ru --quic-drop --fbegin --fend --fbegin --udp-dport-filter=50000-50099 --tls=disabled" из параллельного топика по разблокировке discord
> 
> при старте отчёт по флагам выглядит очень странно жду ответа там

если я правильно понял, то "кастомная часть" должна начинаться с --fbegin и заканчиваться --fend. Во флагах выше ТЕОРЕТИЧЕСКИ первая секция пропадает (которая --sni-domains=all --exclude-domains=.ru --quic-drop), далее пустая секция, и потом начинается секция --udp-dport-filter=50000-50099 --tls=disabled
А такой конфиг у кого-то работает?

**Comment by sg7on** [comment]
> А такой конфиг у кого-то работает?

не знаю)
я так понимаю, что большинство конфигурирует из веб-интерфейса в openwrt

а в кинетике с opkg есть только сам файл со строчкой ARGS, которая изначально пуста (и уже всё работает, в смысле youtube, что намекает, что есть исходный набор настроек)

**Comment by dmikhalsky** [comment]
> > А такой конфиг у кого-то работает?
> 
> не знаю) я так понимаю, что большинство конфигурирует из веб-интерфейса в openwrt
> 
> а в кинетике с opkg есть только сам файл со строчкой ARGS, которая изначально пуста (и уже всё работает, в смысле youtube, что намекает, что есть исходный набор настроек)

Да. Я перешел с Openwrt на Asus c Entware. Пакет явно создавался под openwrt, там все работает идеально
